### PR TITLE
utilize new interventions types

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Breaking
This is a breaking change for simulation, calibration.

# Description
Utilize new intervention type for static interventions
dynamics are not yet touched
I have removed the testing of interventions in simulations for 2 reasons
1) mocking this endpoint may be awkward with the mocking we are doing in the conversion steps.
2) this is not a permanent solution and i dont think its worth spending more time trying to fix this test.

### CODE REVIEW

File: service/models/converters.py
Comment (YohannParis) [inline service/models/converters.py:1]: Looks good, but I'm not sure all those `logging.error` are necessary.
They do not have an `infor` or `warn` instead?

**Review by YohannParis — APPROVED** [review]

File: service/models/converters.py
Comment (Tom-Szendrey) [reply service/models/converters.py:1]: Cleaning those up right now - that was me going crazy with trying to understand the permissions issue

File: service/utils/tds.py [hunk 1]
```diff
@@ -318,3 +319,22 @@ def attach_files(output: dict, job_id, status="complete"):
...
+    intervention_response = tds_session().get(intervention_url)
+    if intervention_response.status_code == 404:
+        raise HTTPException(status_code=404, detail="Intervention not found")
+
+    intervention_path = os.path.join(job_dir, f"./{policy_intervention_id}.json")
+    with open(intervention_path, "w") as file:
+        # Ensure we don't have null observables which can be problematic downstream, if so convert
+        # to empty list
+        intervention_json = intervention_response.json()
+        json.dump(intervention_json, file)
```
Comment (mwdchang) [inline service/utils/tds.py:338]: - What is the purpose of dumping the content into a file? 

- I don't understand the observables comment, you are also not doing anything with observables here

Comment (Tom-Szendrey) [reply service/utils/tds.py:338]: obs comment is leftover from copy paste
I figured dumping into a file was okay for debugging - similar to how we are dumping AMRs into files as well.
If you think we should not I can remove that

Comment (mwdchang) [reply service/utils/tds.py:336]: Debugging file is fine.

**Review by mwdchang — APPROVED** [review]

**Pull request merged**